controllers/api/auth_test.go

```diff
@@ -0,0 +1,283 @@
+package api_controllers_test
+
+import (
+	. "go-google-scraper-challenge/helpers/test"
+	"go-google-scraper-challenge/initializers"
+	"net/http"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+type LoginResponse struct {
+	AccessToken  string `json:"access_token"`
+	ExpiresIn    int    `json:"expires_in"`
+	RefreshToken string `json:"refresh_token"`
+	TokenType    string `json:"token_type"`
+}
+
+type ErrorResponse struct {
```

Comment:
There is a bit a duplication tests for these structs. How about adding those to a new directory `tests/`? Technically those structs are serializer objects (i.e. templates for APIs) so these could go in ``tests/serializers/` 💭

Reply:
@olivierobert Moved these structs to `test/serializers` along with other test helpers on [c1c40a6](https://github.com/carryall/go-google-scraper-challenge/pull/26/commits/c1c40a6ec30cf1c3912fe24ba98409a4e796598e)